Tyrant, probability controls, revised ui

curlymorphic opened the pull request:
![Tyrant](https://user-images.githubusercontent.com/7412852/81865133-711a3580-9565-11ea-9e4b-f1501278fc50.png)

Following discussions with @mattcollier on Discord, probability controls have been added. The UI has also been reworked to use the font suggested, and the logo revised. The readme has been updated, with a short tutorial youtube video linked.

The addition of the probability controls I feel is a very nice addition, greatly increasing the scope of use.

Youtube video 
[![Tutorial](https://img.youtube.com/vi/OJSy8fOxI5I/0.jpg)](https://www.youtube.com/watch?v=OJSy8fOxI5I)

mattcollier commented:
@curlymorphic Really nice video!!!  When asked, Omri says he has some way of ducking the Rack audio under his voice.  That would be a nice thing to figure out for future vids.

Feature idea that came to mind seeing how the accent stuffs work.  Is that it would be cool to have a polyphonic trigger output for all/both accent channels.  Such that when the accent happens, a trigger also occurs.  This could be used *solely* for the purpose of changing an envelope for some other effect, or in combination with CV changes generated by Tyrant.

Regarding the UI.
First, I really like the font for 'Tyrant' label. If we can address the crowding I describe below, I would like to see the module title have a larger top margin.  

I'm not sure how I feel about using that font for everything.  I think I might prefer a san-serif font for labeling knobs and inputs.

Generally the UI is crowded and hard to at a glance determine which labels correspond to which inputs.

I think making the module wider and using a different font will make it feel less crowded.  This would absolutely be required if additional outputs are added.

I think you mentioned this in another forum but I thing the 6+1 logo should be about 20% smaller.

curlymorphic commented:
> Feature idea that came to mind seeing how the accent stuffs work. Is that it would be cool to have a polyphonic trigger output for all/both accent channels. Such that when the accent happens, a trigger also occurs. This could be used _solely_ for the purpose of changing an envelope for some other effect, or in combination with CV changes generated by Tyrant.

This is a nice idea, with the five probability options, this would make a nice expansion module, I shall open an issue.

> 
> Regarding the UI.
> First, I really like the font for 'Tyrant' label. If we can address the crowding I describe below, I would like to see the module title have a larger top margin.
> 
> I'm not sure how I feel about using that font for everything. I think I might prefer a san-serif font for labeling knobs and inputs.
> 
> Generally the UI is crowded and hard to at a glance determine which labels correspond to which inputs.
> 
> I think making the module wider and using a different font will make it feel less crowded. This would absolutely be required if additional outputs are added.
> 
> I think you mentioned this in another forum but I thing the 6+1 logo should be about 20% smaller.

Sure, we can make the module wider, an extra couple of hp spacing down the middle should provide some clarity, The 6+1 logo can be made smaller allowing all the controls to be moved down, giving a larger margin for the top logo, It would be nice if we can get the logo to sit below the rack mount screws. I shall revert the label text as suggested.

curlymorphic commented:
![image](https://user-images.githubusercontent.com/7412852/81949882-3a8efa00-95fb-11ea-8c52-c4e11b1b894b.png)
[PolyShiftRegister.zip](https://github.com/StudioSixPlusOne/rack-modules/files/4629178/PolyShiftRegister.zip)

@mattcollier I have revised the svg, hopefully to your taste. I have not updated the code as I feel it would be best if we can finalize the svg first. I shall refrain from any additional UI work on other modules until this is complete, then I can transfer the principals.

I have included the svg as you have a better idea of what you wish to achieve than me.

mattcollier commented:
100% improvement, ship it! :)

curlymorphic commented:
![image](https://user-images.githubusercontent.com/7412852/81961387-34544a00-960a-11ea-9c8b-c98dbd2efcf4.png)
Updated the ui, screenshot in manual, and added the tutorial patch. I shall update the tutorial video when all the modules for the release are complete. This is now ready for testing.

>Really nice video!!! When asked, Omri says he has some way of ducking the Rack audio under his voice. That would be a nice thing to figure out for future vids.

@mattcollier I already added this request  Omri's list of videos to make, when he asked a couple of weeks ago.  The tutorial video was my first ever attempt at talking on a video, I recorded the video using VCV Record while recording my voice using a standalone Zoom h4n pro recorder, with a windsock, as I have no pop shield, nor a decent mic anymore to plug directly into my sound card. Improvement comes with practice.
